sys/tests/time.c

```diff
@@ -0,0 +1,50 @@
+#include <sys/mimiker.h>
+#include <errno.h>
+#include <sys/ktest.h>
+#include <sys/time.h>
+
+static int test_do_clock_nanosleep(void) {
```

Comment:
Any case where `do_clock_nanosleep` returns something other than EINVAL?

Reply:
Yes, for example: EINTR, but for do_clock_nanosleep we focus only on arguments correctness, the sleep is tested in test_nanosleep.